homeassistant/components/august/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "august",
   "name": "August",
   "documentation": "https://www.home-assistant.io/integrations/august",
-  "requirements": ["py-august==0.7.0"],
+  "requirements": ["py-august==0.8.1"],
```

Comment:
@bdraco looks like py-august 0.10.0 eas released shortly after 0.8
1 (with token auto renewal), might be relevant to bump it here. wdyt?

Reply:
@dannyk81 
While 0.10.0 makes thing nicer with auto token refresh, When I tested 0.10.0, I found it has a blocking bug:
https://github.com/snjoetw/py-august/pull/25

Once it is fixed, it can be done in a new PR.  Ideally this PR focuses on getting the integration working again and we can do improvements in a followup.